src/backends/custom/custom_backend_factory.cc

```diff
@@ -66,12 +66,20 @@ CustomBackendFactory::CreateBackend(
       GetDirectoryFiles(path, true /* skip_hidden_files */, &custom_files));
 
   std::unordered_map<std::string, std::string> custom_paths;
+  std::vector<std::shared_ptr<TemporaryDirectory>> local_custom(
+      custom_files.size());
+  for (size_t s = 0; s < custom_files.size(); s++) {
+    local_custom[s] = std::make_shared<TemporaryDirectory>("");
+  }
+
+  int i = 0;
   for (const auto& filename : custom_files) {
     const auto custom_path = JoinPath({path, filename});
-
+    RETURN_IF_ERROR(LocalizeFileFolder(custom_path, local_custom[i]));
     custom_paths.emplace(
         std::piecewise_construct, std::make_tuple(filename),
-        std::make_tuple(custom_path));
+        std::make_tuple(local_custom[i]->model_path));
+    i++;
```

Comment:
Below why are we using custom_paths instead of local_custom (local_custom_paths would be a better name)? We localized the model, why are we not using the localized version?

Reply:
Fixed naming. custom_paths are updated with the local_custom_paths corresponding to them. Therefore it contains the localized paths and not the true cloud repository path.